### DISCUSSION TITLE
Zigbee2mqtt does not start as service

### DISCUSSION THREAD

**Discussion by andaluzo** [Koenkk/zigbee2mqtt#16928] [General]
I have installed zigbee2mqtt on a raspberrypi it starts and run fine with 
/opt/zigbee2mqtt/npm start 

when I try to start as a service I get in the logs. 

Error: The module '/opt/zigbee2mqtt/node_modules/@serialport/bindings/build/Release/bindings.node' was compiled against a different Node.js version using NODE_MODULE_VERSION 108. This version of Node.js requires
 NODE_MODULE_VERSION 93. Please try re-compiling or re-installing
 
npm -v is 8.19.1
node -v is v18.9.0

I tried already 

rm -rf node_modules && npm install
and 
npm rebuild @serialport/bindings --update-binary

What else could I try ?

**Comment by arteck** [comment]
npm rebuild @serialport

**Reply by andaluzo** [reply]
npm rebuild @serialport

produces an error and 

I tried already 
npm rebuild @serialport/bindings --update-binary

without success 

thanks

**Comment by lux73** [comment]
npm and node Version are fine - did you upgrade nodeJS?

try `npm rebuild` inside z2m Folder

Otherwise backup your /data Directory inside zigbee2mqtt Folder and try completely fresh Installation
